test/functional/Java19andUp/playlist.xml

```diff
@@ -23,6 +23,12 @@
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TestConfig/playlist.xsd">
 	<test>
 		<testCaseName>Jep425Tests_testVirtualThread</testCaseName>
+		<disables>
+			<disable>
+				<comment>https://github.com/eclipse-openj9/openj9/issues/15650</comment>
+				<comment>https://github.com/eclipse-openj9/openj9/issues/15651</comment>
```

Comment:
This will not pass the TKG check. Only one comment is expected.

Reply:
Yeah, removed the second one, and verified locally.